content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -74,10 +74,30 @@ If you define a Job that uses sidecar using Kubernetes-style init containers,
 the sidecar container in each Pod does not prevent the Job from completing after the
 main container has finished.
 
+{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
+
+This feature is also useful for running Jobs with sidecars, as the sidecar
+container will not prevent the Job from completing after the main container
+has finished.
+
 Here's an example of a Job with two containers, one of which is a sidecar:
 
 {{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
 
+### Sidecar containers without Init Containers (Legacy Definition)
+
+Compared to regular defined within `initContainers`, you could 
+directly use multiple `containers` definitions for a Pod, the 
+kubelet starts the containers from the ordered `.spec.containers` list.
+
+Here's an example of a Deployment with two containers:
+
+{{% code_sample language="yaml" file="application/deployment-sidecar-legacy.yaml" %}}
+
+Here's an example of a Job with two containers:
+
+{{% code_sample language="yaml" file="application/job/job-sidecar-legacy.yaml" %}}
```

Comment:
We could use tabs to show the legacy way and also the recommended alternative. WDYT?

Reply:
(if we do, we already have a code sample available I think)

Reply:
See https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs